README.md

```diff
@@ -129,15 +129,15 @@ And that’s it — you should now be able to copy our code samples into your `i
 
 1. Install `Node/npm`. Below is a link to find the install method that coincides with your operating system:
 
-    - Node v4.2.3+, [Installation guides](https://nodejs.org/en/download/)
+    - Node v10.15.1+, [Installation guides](https://nodejs.org/en/download/)
```

Comment:
Do we know if anything above 10 works? e.g. does node 11 work?

Reply:
Untested afaik

Reply:
Updated to be explicitly node 10 in f65e9fb.